Documentation/api.md

```diff
@@ -243,6 +243,7 @@ The `NodeExporterCollectorNetClassConfig` resource works as an on/off switch for
 | -------- | ---- | ----------- |
 | enabled | bool | A Boolean flag that enables or disables the `netclass` collector. |
 | useNetlink | bool | A Boolean flag that activates the `netlink` implementation of the `netclass` collector. This implementation improves the performance of the `netclass` collector by omitting these metrics: `node_network_address_assign_type`, `node_network_name_assign_type`, `node_network_device_id`, `node_network_speed_bytes`. In addition, The `node_network_info` metric lacks the `duplex` label. |
+| ignoredNetworkDevices | *[]string | A list of network devices to be excluded from collection, using regular expressions. By default, the following list is used: `veth.*`, `[a-f0-9]{15}`, `enP.*`, `ovn-k8s-mp[0-9]*`, `br-ex`, `br-int`, `br-ext`, `br[0-9]*`, `tun[0-9]*`, `cali[a-f0-9]*` |
```

Comment:
I wouldn't document the exact list as it would quickly become out of date. Maybe include some general phrasing that by default, node_exporter will try to collect metrics only from physical network interfaces.

Reply:
my intent was that users who override this parameter can easily know what's the setting baseline, so they have a known working starting point when they want to update this setting - but yes this is assuming this doc is always kept up to date. Let me know if you still prefer that I remove it.